apps/estuary/config/integration.exs

```diff
@@ -7,7 +7,7 @@ config :prestige,
   headers: [
     user: "estuary",
     catalog: "hive",
-    schema: "event_stream"
+    schema: "default"
```

Comment:
if you've got this setting in `config.exs` and you _dont_ override it anywhere, you shouldn't need to define it anywhere else, including in `runtime.exs`